sound/soc/codecs/rt715-sdca.c

```diff
@@ -286,7 +288,22 @@ static int rt715_sdca_put_volsw(struct snd_kcontrol *kcontrol,
 		if (err < 0)
 			return err;
 	}
-
+#if IS_ENABLED(CONFIG_DELL_PRIVACY)
+	/* Privacy LED Trigger State Changed by muted/unmute switch */
+	if (mc->invert) {
+		val0 = ucontrol->value.integer.value[0];
+		val1 = ucontrol->value.integer.value[1];
+		if (val0 == 1 && val1 == 1) {
+			rt715->micmute_led = LED_OFF;
+			ledtrig_audio_set(LED_AUDIO_MICMUTE,
+					rt715->micmute_led ? LED_ON : LED_OFF);
+		} else if (val0 == 0 && val1 == 0) {
+			rt715->micmute_led = LED_ON;
+			ledtrig_audio_set(LED_AUDIO_MICMUTE,
+					rt715->micmute_led ? LED_ON : LED_OFF);
```

Comment:
You can use `ledtrig_audio_set(LED_AUDIO_MICMUTE, rt715->micmute_led);`. Actually, I am thinking do we need `rt715->micmute_led`? When will `rt715->micmute_led` be used except ledtrig_audio_set? Can `micmute_led` be a local variable?

Reply:
Hi @bardliao 
Thanks for your review.
Actually the privacy driver  dose not need to input the parameter brightness for EC notification. 
But i don`t want to make the code reviewers confused why i set the brightness without actual value.
`ledtrig_audio_set(LED_AUDIO_MICMUTE, rt715->micmute_led);`
I will change this part like above. it is more simple.